File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/OrderInfoMessage.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class OrderInfoMessage {
    @JsonProperty("stark_key")
    private String starkKey;
    @JsonProperty("contract1")
    private String contract1;
    @JsonProperty("contract2")
    private String contract2;
}

File: reddio-csharp/reddio.api/V1/Rest/OrderInfoMessage.cs
using Newtonsoft.Json;

namespace Reddio.Api.V1.Rest
{
    public class OrderInfoMessage
    {
        public OrderInfoMessage(string starkKey, string contract1, string contract2)
        {
            StarkKey = starkKey;
            Contract1 = contract1;
            Contract2 = contract2;
        }

        public OrderInfoMessage()
        {
        }

        [JsonProperty("stark_key")] public string StarkKey { get; set; }
        [JsonProperty("contract1")] public string Contract1 { get; set; }
        [JsonProperty("contract2")] public string Contract2 { get; set; }
    }
}